back/mailer.js

```diff
@@ -115,32 +175,10 @@ const sendEmail = async (text, emailAddress, emailSettings) => {
       subject: subject,
       text: text,
     });
-    
-    console.log('Message sent: %s', info.messageId);
-
+    console.log('Message sent:', info.messageId);
   } catch (error) {
-    console.error(error);
-  }
-};
-
-const email = async (message, key, opts) => {
-  const emailSettings = await services.emailSettings.read();
-  const emailAddress = await services.user.getEmail(key);
-
-  if (emailSettings.shouldSend !== 'true') {
-    return;
-  }
-
-  switch (message) {
-  case 'assigned':
-  case 'update':
-    if (emailSettings.shouldQueue === 'true') {
-      services.pendingEmails.add(message, key, opts.scheduleId);
-      break;
-    }
-  default:
-    sendEmail(getText(message, opts, emailSettings), emailAddress, emailSettings);
-    break;
+    console.error('Error in sendEmail:', error);
+    throw error;
```

Comment:
It seems that this error is not always caught? Would lead to the app crashing. It seems that the sendEmail function is used without try-catch blocks elsewhere, so perhaps this throw statement should be removed.